lib/diagnostic.js

```diff
@@ -38,14 +59,28 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = (); {
+  name: 'Peter Fluffy Bottom';
+  species: 'Rabbit';
+  breed: 'Whatever rabbits are';
+  vocalization: 'bark';
+//I'm a little less confident now
+  vocalize: function() {
+    let vocalization = this.vocalization;
+    return vocalization;
+  }
+
+};
 
```

Comment:
```js
let pet = {
  name: 'Beau',
  species: 'Dog',
  breed: 'Lab',
  vocalization: 'bark',
  vocalize: function () {
    return this.vocalization;
  }
};
```

looks like you have the right ideas, but need to get a little more familair with the syntax.